interactive/models.py

```diff
@@ -93,3 +101,17 @@ class AnalysisRequest(models.Model):
 
     def __str__(self) -> str:
         return f"{self.title} ({self.codelist})"
+
+
+@receiver(post_save, sender=User)
+def send_email(sender, instance, created, **kwargs):
+    if not created:
+        return
+
+    context = {
+        "name": instance.name,
+        "domain": settings.BASE_URL,
+        "uid": urlsafe_base64_encode(force_bytes(instance.pk)),
+        "token": PasswordResetTokenGenerator().make_token(instance),
+    }
+    send_welcome_email(instance.email, context)
```

Comment:
I think this is probably fine for an MVP but my experience with signals has been nearly entirely negative because debugging them is tricky.  If possible I'd opt to customise a view or build a wrapper to do this.